Cycling Analysis/Strava importer.ipynb

```diff
@@ -348,41 +357,43 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 53,
    "metadata": {},
    "outputs": [],
    "source": [
-    "def max_power(data, max_durations):\n",
```

Comment:
The user inputs should be properly validated to prevent injection attack or unexpected behaviour.Validating data types where appropriate

Reply:
Hi @pranav5722222, the inputs for this function are not user supplied. They are created by functions that we have defined that read in .fit files and if the .fit file has incorrect data then the power fields will be set to zero.